metaflow/metadata/heartbeat.py

```diff
@@ -4,7 +4,7 @@
 
 from threading import Thread
 from metaflow.sidecar import MessageTypes, Message
-from metaflow.metaflow_config import METADATA_SERVICE_HEADERS
```

Comment:
[NIT]: what was the problem with `METADATA_SERVICE_HEADERS`? Not a blocker but just curious.

Reply:
The external user-facing env name was METAFLOW_SERVICE_HEADERS so I standardized to that (kind of needed to make it work :( ).

Reply:
Can you add some color on why `METAFLOW_` needs to be stripped out for what you intend to achieve?